code/modules/religion/sects/shadow_sect.dm

```diff
@@ -394,18 +375,36 @@
 	lisk.AddComponent(/datum/component/dark_favor, user)
 	lisk.set_light(sect.light_reach, sect.light_power, DARKNESS_INVERSE_COLOR)
 	playsound(altar_turf, 'sound/magic/fireball.ogg', 50, TRUE)
-	priority_announce("May our lord, [GLOB.deity], have mercy on your soul as darkness reigns apon you all.", "Faith Alert", SSstation.announcer.get_rand_alert_sound())
-	addtimer(CALLBACK(lisk, TYPE_PROC_REF(/obj/structure/destructible/religion/shadow_obelisk, final_darkness_activate)), 100)
+	if(user.mind.is_murderbone())
+		priority_announce("May our lord, [GLOB.deity], have mercy on your soul as darkness reigns apon you all.", "Faith Alert", SSstation.announcer.get_rand_alert_sound())
+	else
+		priority_announce("May our lord, [GLOB.deity], bless all the shadows as darkness heals those who worship the night.", "Faith Alert", SSstation.announcer.get_rand_alert_sound())
+	addtimer(CALLBACK(lisk, TYPE_PROC_REF(/obj/structure/destructible/religion/shadow_obelisk, final_darkness_activate), user), 250)
 
-/obj/structure/destructible/religion/shadow_obelisk/proc/final_darkness_activate()
+/obj/structure/destructible/religion/shadow_obelisk/proc/final_darkness_activate(mob/living/user)
 	var/datum/religion_sect/shadow_sect/sect = GLOB.religious_sect
+	var/list/candidates = poll_ghost_candidates("Do you wish to be summoned as a Shadow Faithful?", ROLE_HOLY_SUMMONED, null, 10 SECONDS, POLL_IGNORE_HOLYUNDEAD)
 	for(var/obj/structure/destructible/religion/shadow_obelisk/obs in sect.obelisks)
 		START_PROCESSING(SSobj, obs)
+		if(!user.mind.is_murderbone())
+			playsound(obs, 'sound/magic/fireball.ogg', 50, TRUE)
+			return
 		var/obelisk_turf = get_turf(obs)
-		for(var/i in 1 to 3)
-			var/mob/living/simple_animal/hostile/faithless/faithful/faithful = new(obelisk_turf)
-			faithful.AddComponent(/datum/component/dark_favor, faithful)
-			faithful.set_light(2, -2, DARKNESS_INVERSE_COLOR)
+		if(prob(30))
+			for(var/i in sect.min_summon to sect.max_summon)
+				var/mob/living/simple_animal/hostile/faithless/faithful/faithful = new(obelisk_turf)
+				faithful.AddComponent(/datum/component/dark_favor, faithful)
+				faithful.set_light(2, -2, DARKNESS_INVERSE_COLOR)
+				if(length(candidates))
```

Comment:
This is how I'd do it, same idea as above. Then move the play sound thing at the bottom to here.
```suggestion
                playsound(obs, 'sound/hallucinations/wail.ogg', 50, TRUE)
				if(!length(candidates))
				    continue
```

Reply:
![image](https://github.com/user-attachments/assets/490fc4a5-6401-4646-a5ea-9e41100f06d8)

This look right?

Reply:
Sure, just don't forget this-
```
if(!length(candidates))
				    continue
				    ```

Reply:
![image](https://github.com/user-attachments/assets/951faa94-7f18-45f6-a444-e6def101c779)

?